src/drivers/block_dev/partition/partition.c

```diff
@@ -23,7 +23,7 @@ int create_partitions(struct block_dev *bdev) {
 	struct mbr mbr;
 	int rc;
 	int part_n;
-	char part_name[0x16];
+	char part_name[50];
```

Comment:
It's about compiler error:
`src/drivers/block_dev/partition/partition.c:45:26: error: ‘%s’ directive writing up to 31 bytes into a region of size 16 [-Werror=format-overflow=]`